cmd/run.go

```diff
@@ -94,3 +96,33 @@ func formatHeadersForConfig(headers []string, token string) map[string]string {
 	}
 	return res
 }
+
+// save result in a file
+func saveResultInFile(result model.ScenarioResult) {
+
+	if len(outputFile) == 0 {
+		return
+	}
+
+	var s []byte
+	var err error
+	// Marshal in the choosing format
+	if strings.ToUpper(outputFormat) == "YAML" || strings.ToUpper(outputFormat) == "YML" {
+		s, err = yaml.Marshal(result)
+	} else {
+		s, err = json.Marshal(result)
+	}
+	check(err)
+
+	err = ioutil.WriteFile(outputFile, s, 0644)
+	check(err)
+
+	logrus.Infof("Output file %s saved", outputFile)
+}
+
+func check(err error) {
```

Comment:
I find check is not explicit name for what the function does. Maybe `exitIfErr` is better name and declare it util package is more appropriate.

Reply:
👍